### PR TITLE
Ubuntu18

### PR DESCRIPTION
**Pull request by volzkzg**
Use ubuntu 18.04 as base image and configure the environment for python3 and rust 2018-05-17

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -21,36 +21,37 @@ RUN apt-get update && apt-get install -y \
...
     cmake \
     build-essential \
- && curl -o v33.tar.gz -L https://github.com/SimonKagstrom/kcov/archive/v33.tar.gz \
- && tar -xf v33.tar.gz && cd kcov-33 \
+    automake \
+    libtool \
+    libffi-dev \
+    libgmp-dev \
+    libyaml-cpp-dev \
+ && curl -o v35.tar.gz -L https://github.com/SimonKagstrom/kcov/archive/v35.tar.gz \
```
Comment (TheWaWaR) [inline Dockerfile:29]: Have you tested kcov-35? As I know kcov-34 have some bugs so I didn't use it.

**Comment by TheWaWaR** [comment]
Shall we create a branch ubuntu1604 first?

**Comment by volzkzg** [comment]
@TheWaWaR I guess we can fork the master branch to ubuntu-16.04-20180519

**Comment by volzkzg** [comment]
@TheWaWaR  And I think right now there is no need to merge this PR. Should we close this PR?

**Comment by TheWaWaR** [comment]
@volzkzg Agreed, no need.

**Pull request closed without merging**